cmd/controller/main.go

```diff
@@ -58,21 +54,18 @@ func main() {
 		LeaderElection:         true,
 		LeaderElectionID:       "karpenter-leader-election",
 		Scheme:                 scheme,
-		Port:                   options.WebhookPort,
 		MetricsBindAddress:     fmt.Sprintf(":%d", options.MetricsPort),
 		HealthProbeBindAddress: fmt.Sprintf(":%d", options.HealthProbePort),
 	})
 
 	clientSet := kubernetes.NewForConfigOrDie(manager.GetConfig())
-	cloudProviderFactory := registry.NewFactory(cloudprovider.Options{Client: manager.GetClient(), ClientSet: clientSet})
+	cloudProvider := registry.NewCloudProvider(cloudprovider.Options{ClientSet: clientSet})
 
-	err := manager.RegisterWebhooks(
```

Comment:
No, actually. The new validation logic registers supported properties on startup (checkout the Register() function), rather than by API call. Even if we did it via API calls, this validation logic only happens on creation/modification of provisioners, which is very rare.